sdk/core/azure_core_tracing_opentelemetry/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "azure_core_tracing_opentelemetry"
+version = "0.1.0"
+description = "OpenTelemetry tracing integration for the Azure SDK for Rust"
+readme = "README.md"
+authors.workspace = true
+license.workspace = true
+repository.workspace = true
+homepage = "https://github.com/azure/azure-sdk-for-rust"
+documentation = "https://docs.rs/azure_core_tracing_opentelemetry"
+keywords = ["sdk", "azure", "tracing", "opentelemetry", "observability"]
+categories = ["api-bindings", "development-tools::debugging"]
+
```

Comment:
Was it your intent to ship this yet? If not, I would add `publish = false` until we're ready. And if you did want to ship it, make sure it's in `sdk/core/ci.yml`.